src/launchoptions.rs

```diff
@@ -0,0 +1,211 @@
+use std::{
+    fs,
+    fs::{File, OpenOptions},
+    io::{Read, Write},
+    path::PathBuf,
+};
+
+use anyhow::{Context, Result};
+use regex::{Match, Regex};
+use steamid_ng::SteamID;
+use substring::Substring;
+use tracing::Level;
+
+use crate::gamefinder::locate_steam_launch_configs;
+
+pub const TF2_GAME_ID: &str = "440";
+pub const CONDEBUG: &str = "-condebug";
+pub const CONCLEARLOG: &str = "-conclearlog";
+pub const USERCON: &str = "-usercon";
+pub const G15: &str = "-g15";
+pub fn get_required_args_vec() -> Vec<String> {
```

Comment:
This doesn't need to be pub

Reply:
Nor do all of the launch options strings that are only used inside this module.

Reply:
Is there a reason these are all put into a `Vec<String>` and stored in the `LaunchOptionsV2` struct? It looks like once they are stored in the struct they are only read from, so you could just put them in a `const REQUIRED_ARGS: [&str; 5] = [...]` and iterate over that global constant rather than reading them from the struct.